examples/ssr/package.json

```diff
@@ -7,6 +7,9 @@
     "start": "razzle start",
     "start:prod": "NODE_ENV=production node build/server.js"
   },
+  "resolutions": {
+    "mini-css-extract-plugin": "^1.6.2"
```

Comment:
Needed to resolve deprecation warnings caused by `"mini-css-extract-plugin": "0.9.0"`